package/native-package/src/index.js

```diff
@@ -247,6 +249,26 @@ registerNativeHandlers({
       ignoreAndroidSystemSettings: false,
     });
   },
+  // eslint-disable-next-line react/display-name
+  Video: ({ onBuffer, onEnd, onLoad, onProgress, paused, resizeMode, style, uri, videoRef }) => (
+    <Video
+      onBuffer={onBuffer}
+      onEnd={onEnd}
+      onError={(error) => {
+        console.log(error);
```

Comment:
Should we log this with `console.error` instead?

Reply:
Yes, we can.